src/Fantomas.Core/CodePrinter.fs

```diff
@@ -1834,15 +1828,36 @@ let genMultilineFunctionApplicationArguments (argExpr: Expr) =
     | _ -> genExpr argExpr
 
 let genTuple (node: ExprTupleNode) =
+    let wrapInfixAppRhsInParenIfNeeded expr =
+        match expr with
+        | Expr.InfixApp exprInfixAppNode ->
+            match exprInfixAppNode.RightHandSide with
+            | e when isSynExprLambdaOrIfThenElse e ->
+                let parenNode =
+                    ExprParenNode(
```

Comment:
Check the usage for ExprPareNode, we already use this in CodePrinter, time to extract a helper for this:
![image](https://user-images.githubusercontent.com/2621499/229800551-1fa17388-89f6-4b1e-b34e-14e1c4cb3ff3.png)